substratevm/src/com.oracle.svm.jfr/src/com/oracle/svm/jfr/traceid/JfrTraceIdMap.java

```diff
@@ -66,6 +66,7 @@ long getId(int index) {
         return traceIDs[index];
     }
 
+    @Uninterruptible(reason = "Called from uninterruptible code.", mayBeInlined = true)
```

Comment:
The call chain to this goes through `SubstrateJVM.getClassId`. This is currently called by two methods, `Target_jdk_jfr_internal_JVM.getClassIdNonIntrinsic` and `JfrNativeEventWriter.putClass`. The second one is marked uninterruptible. Could you also mark `SubstrateJVM.getClassId` as uninterruptible + mayBeInlined = true in this PR as well? It'll prevent a future bug when native events through `JfrNativeEventWriter` try to `putClass`.

Reply:
Done. I needed to add some more annotations to satisfy the compiler. ;-)

Reply:
So many uninterruptible methods D: heh. Thanks!